### ISSUE TITLE
Multiple Vue instances causing problems

### ISSUE THREAD

**Issue by jmkenz** [richardtallent/vue-simple-calendar#106]
Hello, and thank you for releasing this to the community!

I've run into a problem where importing this into my app is importing a 2nd instance of Vue and causing conflicts.  (A similar issue is described here: https://github.com/euvl/vue-js-modal/issues/21)

I believe this can be resolved by simplifying your bundle.js to avoid registering the components on a Vue instance, and simply exporting the components, leaving it to end users to register the components with their app's instance of Vue.

For now I've been able to work around this by importing like so:

```
import CalendarView from 'vue-simple-calendar/src/components/CalendarView';
import CalendarViewHeader from 'vue-simple-calendar/src/components/CalendarViewHeader';
```

**Comment by richardtallent** [comment]
Interesting! This sounds like it could be a breaking change for the majority of users, does anyone else have any feedback?

**Comment by samitnuk** [comment]
@richardtallent 

Hello!

I have similar issue with `vue-simple-calendar`.

With
```
import { CalendarView, CalendarViewHeader } from "vue-simple-calendar"
```
I have constant repetition of the same warnings in the browser's console (E.g. on calendar loading, hovering an event with cursor etc.):
```
[Vue warn]: $attrs is readonly.
[Vue warn]: $listeners is readonly.
[Vue warn]: Avoid mutating a prop directly since the value will be overwritten whenever the parent component re-renders. ...
```
I found that these warnings caused of two different instances of `Vue` in the bundle.

But as @jmkenz mentioned, importing from `*.vue` files fixed this.
```
import CalendarView from "vue-simple-calendar/src/components/CalendarView.vue"
import CalendarViewHeader from "vue-simple-calendar/src/components/CalendarViewHeader.vue"
```

**Comment by richardtallent** [comment]
Interesting. Importing directly from `*.vue` files should be avoided, since this usually means you'll be bypassing Babel for the calendar's code and creating browser compatibility issues (Babel doesn't transform code in `node_modules`).

I haven't dealt with having multiple Vue instances before, but if removing the registration in `plugin.js` solves the problem without creating a significant upgrade headache or new issues for other users (users with one Vue instance, people using Nuxt, etc.), I'm good with that change.

**Comment by rickyruiz** [comment]
I just ran into this issue when updating to the latest Vue version (2.5.21), is there a status update for this issue?

**Comment by richardtallent** [comment]
Update -- this will be fixed in 4.2.0, coming very soon... just need to do a little testing in Edge for another issue before I publish it.

**Comment by richardtallent** [comment]
Resolved in 4.2.0, just published. Please re-open if this doesn't fix the issue.

**Comment by rickyruiz** [comment]
@richardtallent It did fix the issue for me. Thanks a lot!
